### PR TITLE
aliases jupyterlab for ease of use

### PR DESCRIPTION
**Pull request by jjgmckenzie**

### CODE REVIEW

File: start-jupyterlab.sh [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+# Check if the directory already exists
+if [ ! -d "$JUPYTERLAB_SCRIPT_DIR/$VENV_NAME" ]; then
+  _pretty_print "Virtual environment does not exist; creating"
+  $JUPYTERLAB_SCRIPT_DIR/create-venv.sh
+fi
+
+source $JUPYTERLAB_SCRIPT_DIR/$VENV_NAME/bin/activate
+
+if which "jupyter" > /dev/null 2>&1; then
+    _pretty_print 'jupyter exists'
```
Comment (jjgmckenzie) [inline start-jupyterlab.sh:18]: Can remove this

File: README.md [hunk 2]
```diff
@@ -1,5 +1,6 @@
 # jupyterlab-ubuntu-local
 Toby's JupyterLab deployment designed for use on a non-web facing Ubuntu instance.
+Requirements: zsh
```
Comment (jjgmckenzie) [inline README.md:3]: Include other requirements

Comment (tobkin) [reply README.md:3]: Remove and add to README in molecule-semantic-search

File: symlink-jupyterlab.sh [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    sed -i "s#export JUPYTERLAB_SCRIPT_DIR=.*#export JUPYTERLAB_SCRIPT_DIR=\"$(pwd)\"#" ~/.zshrc
+  else # If the variable is not exported, add it to the end of the file
+    echo "export JUPYTERLAB_SCRIPT_DIR=\"$(pwd)\"" >> ~/.zshrc
+fi
+
+# Check if ~/bin directory exists, create it if not
+if [ ! -d ~/bin ]; then
+  mkdir -p ~/bin
+  # Check if the directory already exists in PATH
+  if [[ ":$PATH:" == *":$HOME/bin:"* ]]; then
```
Comment (tobkin) [inline symlink-jupyterlab.sh:22]: Unnest

**Review by tobkin — APPROVED** [review]

**Pull request merged**